### PR TITLE
feat(icons): add icon instagram

### PR DESCRIPTION
**Pull request by gabrielduete**
## Infos

[Task](https://juntossomosmais.monday.com/boards/XXX/pulses/XXX)

#### What is being delivered?

Add instagram icon in MDI icons.

#### What impacts?

Icons

#### Reversal plan

Revert PR

#### Evidences

image

### CODE REVIEW

File: packages/icons/svg/mdi/instagram.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
+<path d="M16.1235 2.06282C14.2835 1.97699 9.72603 1.98116 7.88436 2.06282C6.26602 2.13866 4.83852 2.52949 3.68769 3.68032C1.76436 5.60366 2.01019 8.19532 2.01019 11.9962C2.01019 15.8862 1.79352 18.4178 3.68769 20.312C5.61852 22.242 8.24769 21.9895 12.0035 21.9895C15.8569 21.9895 17.1869 21.992 18.5494 21.4645C20.4019 20.7453 21.8002 19.0895 21.9369 16.1153C22.0235 14.2745 22.0185 9.71782 21.9369 7.87616C21.7719 4.36532 19.8877 2.23616 16.1235 2.06282ZM19.036 19.0395C17.7752 20.3003 16.026 20.1878 11.9794 20.1878C7.81269 20.1878 6.14186 20.2495 4.92269 19.027C3.51852 17.6295 3.77269 15.3853 3.77269 11.9828C3.77269 7.37865 3.30019 4.06282 7.92102 3.82616C8.98269 3.78866 9.29519 3.77616 11.9677 3.77616L12.0052 3.80116C16.446 3.80116 19.9302 3.33616 20.1394 7.95615C20.1869 9.01032 20.1977 9.32699 20.1977 11.9953C20.1969 16.1137 20.2752 17.7945 19.036 19.0395Z" fill="#1F2328"/>
+<path d="M17.3387 7.8607C18.001 7.8607 18.5379 7.32382 18.5379 6.66154C18.5379 5.99926 18.001 5.46237 17.3387 5.46237C16.6764 5.46237 16.1395 5.99926 16.1395 6.66154C16.1395 7.32382 16.6764 7.8607 17.3387 7.8607Z" fill="#1F2328"/>
```
Comment (igorwessel) [inline packages/icons/svg/mdi/instagram.svg:4]: **nitpick:** ‏we need to remove fill to allow change colors

**Review by igorwessel — CHANGES_REQUESTED** [review]

File: packages/icons/svg/mdi/instagram.svg (hunk 1 above)
Comment (gabrielduete) [reply packages/icons/svg/mdi/instagram.svg]: I need to use the current color to be able to change colors

image
image


Made: https://github.com/juntossomosmais/atomium/pull/506/commits/84ba3c4e0ed9236f6de34ea571aca1b45ca0b598

Comment (igorwessel) [reply packages/icons/svg/mdi/instagram.svg]: you can also remove `fill` attribute.

**Review by igorwessel — APPROVED** [review]

**Review by felipefialho — DISMISSED** [review]

File: packages/icons/svg/mdi/instagram.svg (hunk 1 above)
Comment (gabrielduete) [reply packages/icons/svg/mdi/instagram.svg]: made: https://github.com/juntossomosmais/atomium/pull/506/commits/dd8a36e7684df6ac02b75519e3a4ce2c18592f06

**Review by tassioFront — APPROVED** [review]

**Review by felipefialho — APPROVED** [review]

**Pull request merged**